docs/accumulo-export-queue.md

```diff
@@ -19,8 +19,8 @@ limitations under the License.
 ## Background
 
 The [Export Queue Recipe][1] provides a generic foundation for building export mechanism to any
-external data store. The [AccumuloExporter] provides an implementation of this recipe for
-Accumulo. The [AccumuloExporter] is located the `fluo-recipes-accumulo` module and provides the
+external data store. The [AccumuloConsumer] provides an export consumer for writing to
```

Comment:
A bit more background.   A lot of these changes were motivated by the desire to allow using lambdas after changes in Fluo allowed this.   To support lambdas, the following happened.

 * Exporter was an abstract class and could not support lambdas, so was deprecated and replaced with ExportConsumer.  ExportConsumer is a functional interface and lambdas can be assigned to it.
 * AccumuloExporter extended Exporter and was deprecated.  It was replaced with AccumuloConsumer (which implements ExportConsumer) and AccumuloTranslator.  AccumuloTranslator is functional interface and therefore lambdas can be assigned to it.

I really like the names Exporter and AccumuloExporter, but they can't support lambdas.